arroyo/processing/strategies/produce.py

```diff
@@ -101,7 +98,7 @@ def join(self, timeout: Optional[float] = None) -> None:
 
             future.result(remaining)
 
-            offset = {message.partition: Position(message.offset, message.timestamp)}
```

Comment:
Was this committing the right offset ?

Reply:
No, i actually messed it up earlier. Trying to stamp that out with position_to_commit